index.bs

```diff
@@ -4548,15 +4548,22 @@ and similarly, closing or aborting the <code>writable</code> side will implicitl
 
 <h2 id="conventions" class="no-num">Conventions</h2>
 
-This specification uses algorithm conventions very similar to those of [[!ECMASCRIPT]]. However, it deviates in the
-following ways, mostly for brevity. It is hoped (and vaguely planned) that eventually the conventions of ECMAScript
-itself will evolve in these ways.
+This specification uses algorithm conventions very similar to those of [[!ECMASCRIPT]], whose rules should be used to
+interpret it (apart from the exceptions enumerated below). In particular, the objects specified here should be treated
+as <a href="https://tc39.github.io/ecma262/#sec-ecmascript-standard-built-in-objects">built-in objects</a>. For example,
+their <code>name</code> and <code>length</code> properties are derived as described by that specification, as are the
+default property descriptor values.
+
```

Comment:
I think it would be nice to explicitly mention the fact that missing and surplus arguments are treated as per ECMASCRIPT function conventions. Although this is implied, it clearly surprises some people.